tests/Security/Permission/DeleteUiTPASPlaceVoterTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Security\Permission;
+
+use CultuurNet\UDB3\ReadModel\DocumentDoesNotExist;
+use CultuurNet\UDB3\ReadModel\DocumentRepository;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+use CultuurNet\UDB3\Role\ValueObjects\Permission;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Ramsey\Uuid\Uuid;
+
+class DeleteUiTPASPlaceVoterTest extends TestCase
+{
+    /** @var DocumentRepository&MockObject */
+    private $documentRepository;
+    private array $uitpasLabels;
+    private string $userId;
+
+    protected function setUp(): void
+    {
+        $this->documentRepository = $this->createMock(DocumentRepository::class);
+        $this->uitpasLabels = ['UiTPAS', 'Paspartoe'];
+        $this->userId = Uuid::uuid4()->toString();
+    }
+
+    public function test_is_valid_success(): void
```

Comment:
The names of the test methods are rather cryptic, it should me something like "deleting a place without UiTPAS labels is allowed"

Reply:
We still mark all the tests with a header doc instead of the `test` prefix, maybe best to stay consistent?

Reply:
@grubolsch what do you think about this remark

> We still mark all the tests with a header doc instead of the test prefix, maybe best to stay consistent?

Reply:
I know I am not doing it consistent, I have not being this for a while.
I really prefer not to use docblocks when i can, this seems to be a more elegant solution to me.

So I prefer to keep it this way, but if really pressed I will change it.